server/lib/actions_controllers.js

```diff
@@ -38,14 +39,15 @@ module.exports = controllers => {
     try {
       if (sanitization) {
         const params = sanitize(req, res, sanitization)
+        await verifySignedReq(req, res)
```

Comment:
Alternatively, the call to `verifySignature` could be moved to controllers themselves, like controllers that throw `unauthorizedApiAccess` errors

Reply:
after playing with the outbox, its nice to regroup all signature request validation in one place, but it looks like controllers can handle it themselves

Reply:
what about having the, now on master, `controllerWrapper` function accept `access=signed`, which would allow to finely set which endpoint should be signed:
```js
// server/controllers/activitypub/activitypub.js could look like this
module.exports = {
  get: ActionsControllers({
    public: {
      // Btw, that endpoint actually doesn't really need to be signed, does it?
      // It doesn't write anything and only returns public data, right?
      actor: require('./actor')
    }
  }),
  post: ActionsControllers({
    signed: {
      inbox: require('./inbox')
    }
  })
}
```